frontend/javascripts/oxalis/default_state.ts

```diff
@@ -50,6 +50,7 @@ const defaultState: OxalisState = {
   datasetConfiguration: defaultDatasetViewConfigurationWithoutNull,
   userConfiguration: {
     autoSaveLayouts: true,
+    autoRenderMeshInProofreading: false,
```

Comment:
was unsure which default makes sense

Reply:
true would be better, as this was the old default :)